### PR TITLE
Add mathconstants to std lib

### PR DESCRIPTION
**Pull request by juancarlospaco**
- Improve Mathematical constants of `math` module.
- All static `const` with documentation and links to Wikipedia and oeis.org, NEP1 Ok.

Mathematical constants are values that are never going to change and can be static on std lib,
and [the `math` module already have a few,](https://github.com/nim-lang/Nim/blob/devel/lib/pure/math.nim#L101-L116) so I try to improve them, with Docs and Links.

:slightly_smiling_face:

### CODE REVIEW

File: lib/pure/math.nim [hunk 1]
```diff
@@ -114,6 +114,7 @@ const
                                            ## after the decimal point
                                            ## for Nim's ``float`` type.
   RadPerDeg = PI / 180.0 ## Number of radians per degree
+include mathconstants  # File containing only mathematical constants.
```
Comment (mratsim) [inline lib/pure/math.nim]: wouldn't it be better to have that as it's own module?

Comment (juancarlospaco) [reply lib/pure/math.nim]: I dunno, maybe is better, what would be the name :grey_question: :thinking:

**Comment by krux02** [comment]
Sorry, but from me this gets a No. Their use case is far too little to add them all. The biggest effects this PR will have is, it will clutter the documentation of the math module with symbols you don't need.

**Comment by juancarlospaco** [comment]
I moved it out of `math` module.

**Comment by krux02** [comment]
Why did you implement them? Did anybody need them for anything?

**Comment by Araq** [comment]
Sorry, but please make this a Nimble package first and if it becomes popular, we can consider to move it to the stdlib.

**Comment by juancarlospaco** [comment]
They are universal constant so they are never going to change, so no additional maintenance required.

As I said, needed some, and the math module seems lacking and I think of improving it,
things I try to send are ISO Standards or standard constants, not random variables.

**Pull request closed without merging**